masschallenge/django-accelerator
# [AC-9477]: Create StartupUpdate edit view

[AC-9477](https://masschallenge.atlassian.net/browse/AC-9477)

How to test:
- Naviagte to http://localhost:8181/startup/20/update/
- Ensure that all the definition of done on [AC-9477](https://masschallenge.atlassian.net/browse/AC-9477)
 are met

Sibling PR: https://github.com/masschallenge/accelerate/pull/3244
